.gitpod.yml

```diff
@@ -6,7 +6,8 @@ tasks:
   - init: |
       ddev start -y
       ddev composer install
-      ddev drush si -y --account-pass=admin --site-name='ddev_gitpod' demo_umami
+      ddev drush si -y --account-pass=admin --site-name='ddev_gitpod' standard
+      ddev drush en lupus_decoupled_ce_api lupus_decoupled_cors lupus_decoupled_menu -y
```

Comment:
tested in gitpod and these modules are enabled but not the main on  - lupus_decoupled

Can we change drush command and enable only  lupus_decoupled and let lupus_decoupled_* enable automatically